Release note for change schedules

tashonrp opened the pull request:
### Context
Added detailed release notes for the updated change-schedules endpoint in the Register ECTs sandbox, including validation rules and changes in behavior for lead providers.

### Guidance to review
I'll need to update this once I've received the validation error messages from Ross. The delay is my fault - I completely forgot to follow up with him on this.

tashonrp committed:
Release note for change schedules

Added detailed release notes for the updated change-schedules endpoint in the Register ECTs sandbox, including validation rules and changes in behavior for lead providers.

tashonrp committed:
Updated change schedules copy

Added example validation errors for participant schedule changes.

tashonrp reviewed (commented):
@claire-hughez Updated version with validation errors included.

claire-hughez commented on app/views/api/release_notes/release_notes.yml:
```diff
@@ -20,7 +20,61 @@
+
+    ## What’s changed from the existing ‘Manage training for early career teachers’ API?
+
+    ### Only the most recent lead provider can update a participant’s schedule or cohort
```
I know it feels like it's shortening it but it's also that it needs to be the CURRENT lead provider as specified in the spec.

For example, if Lead Provider 1 is training Bob, and then Lead Provider 2 is taking over on the 13th December, neither can change the schedule until Bob starts with LP2 on the 13th December.

claire-hughez commented on app/views/api/release_notes/release_notes.yml:
````diff
@@ -20,7 +20,61 @@
+    **Schedule change would invalidate declarations**
+    ```
+    The schedule change cannot be applied because it would make existing declarations invalid. Please contact DfE for assistance.
+    ```
````
The change of schedule cannot be applied because a previous change of schedule and a declaration were made on the same day. Applying another change of schedule would invalidate existing declarations. Please contact DfE for assistance.

tashonrp committed:
Revision to change schedules release note following feedback

Updated release notes for the schedules endpoint with new validation rules and clarified language regarding lead provider responsibilities.

claire-hughez commented on app/views/api/release_notes/release_notes.yml:
```diff
@@ -20,7 +20,72 @@
 
+    Previously, billable declarations prevented any further changes.
+
+    ### You cannot make multiple cohort changes on the same day when submitting a declaration
```
'You cannot make multiple cohort changes on the same after after submitting a declaration'

claire-hughez commented on app/views/api/release_notes/release_notes.yml:
```diff
@@ -20,7 +20,72 @@
 #
 #     **bold** and _italic_ text
+- title: Change schedules endpoint released to the Register ECTs sandbox with updated validation rules
+  date: 2025-12-XX
```
It's dependent on when declarations is done, so whenever we merge this because it's all finished

tashonrp replied:
Will set to 2026-01-16.

tashonrp committed:
Update release notes for schedules endpoint changes

Updated release notes for the schedules endpoint and adjusted validation rules. Modified the date for the release and clarified restrictions on schedule and cohort changes after a declaration is submitted.

tashonrp committed:
Fix formatting for training completion message
